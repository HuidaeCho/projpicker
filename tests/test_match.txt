#geoms: postfix match_max=1 33.7490,-84.3880 xy 432697.24,1363705.31 match
#python
# import sys
# sys.path.insert(0, "../projpicker")
# import projpicker as ppik
# bbox = ppik.query_mixed_geoms(["postfix", "match_max=1", [33.7490, -84.3880], "xy", [432697.24, 1363705.31], "match"])
# ppik.print_bbox(bbox)
#end

postfix
match_max=1
33.7490,-84.3880
xy
432697.24,1363705.31
match
