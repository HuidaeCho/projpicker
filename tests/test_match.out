proj_table|crs_name|crs_auth_name|crs_code|usage_auth_name|usage_code|extent_auth_name|extent_code|south_lat|north_lat|west_lon|east_lon|bottom|top|left|right|unit|area_sqkm
projected_crs|NAD27 / Georgia West|EPSG|26767|EPSG|6602|EPSG|2190|30.62|35.01|-85.61|-82.99|227321.736222316|1825636.8909584181|45969.582735703174|870089.0814069586|US foot|119521.02819197961
